packages/test-helpers/src/configuration.rs

```diff
@@ -156,3 +156,15 @@ pub fn ephemeral_with_no_services() -> Configuration {
 
     cfg
 }
+
+/// Ephemeral with only udp
+#[must_use]
+pub fn ephemeral_udp_only() -> Configuration {
+    let mut cfg = ephemeral();
+
+    cfg.http_api.enabled = false;
+    cfg.http_trackers[0].enabled = false;
+    cfg.udp_trackers[0].enabled = false;
```

Comment:
Hi @da2ce7 should this line `cfg.udp_trackers[0].enabled = false;` be `cfg.udp_trackers[0].enabled = true;`?

Reply:
Fixed :+1: